### PR TITLE
Settlement for paygops

### PR DESCRIPTION
**Pull request by truthfool**
- Settlement for paygops and roster
@fynmanoj @ankita10r

### CODE REVIEW

File: src/main/resources/application.yml [hunk 1]
```diff
@@ -22,11 +22,11 @@ zeebe:
 timer: "PT30S"
 
 channel:
-  host: "http://localhost:5001"
+  host: http://ph-ee-connector-channel
```
Comment (fynmanoj) [inline src/main/resources/application.yml:25]: Keep these values default

**Review by fynmanoj — CHANGES_REQUESTED** [review]

File: src/main/resources/application.yml (hunk 1 above)
Comment (truthfool) [reply src/main/resources/application.yml:25]: ok

File: src/main/java/org/mifos/connector/mpesa/utility/MpesaUtils.java [hunk 2]
```diff
@@ -31,6 +32,31 @@ public class MpesaUtils {
...
+            partyIdInfoPayee.put("partyIdentifier", paybillConfirmationRequestDTO.getBillRefNo());
+        } else if (amsName.equalsIgnoreCase("roster")) {
+            partyIdInfoPayee.put("partyIdType", "ACCOUNTID");
+            partyIdInfoPayee.put("partyIdentifier", paybillConfirmationRequestDTO.getBillRefNo());
+        }
+        payee.put("partyIdInfo", partyIdInfoPayee);
+
+        JSONObject amount = new JSONObject();
+        amount.put("amount", paybillConfirmationRequestDTO.getTransactionAmount());
+        amount.put("currency", "TZS");
```
Comment (fynmanoj) [inline src/main/java/org/mifos/connector/mpesa/utility/MpesaUtils.java:55]: Currency should not be hardcoded.

Comment (truthfool) [reply src/main/java/org/mifos/connector/mpesa/utility/MpesaUtils.java:55]: we don't receive currency as part of C2B mpesa payload.

Comment (fynmanoj) [reply src/main/java/org/mifos/connector/mpesa/utility/MpesaUtils.java:55]: can we populate from the webhook url ?

Comment (truthfool) [reply src/main/java/org/mifos/connector/mpesa/utility/MpesaUtils.java]: added it from params

File: src/main/java/org/mifos/connector/mpesa/camel/routes/PaybillRoute.java [hunk 3]
```diff
@@ -50,14 +54,14 @@ public void configure() {
                     PaybillRequestDTO paybillRequestDTO = e.getIn().getBody(PaybillRequestDTO.class);
                     //Getting the ams name
                     String amsName = e.getIn().getHeader("amsName", String.class);
-                    logger.info("AMS Name : {}", amsName);
+                    String currency = e.getIn().getHeader("currency", String.class);
+                    logger.debug("AMS Name : {}", amsName);
```
Comment (fynmanoj) [inline src/main/java/org/mifos/connector/mpesa/camel/routes/PaybillRoute.java:58]: AMS name design needs to be updated

**Pull request merged**